files/default/slurm/templates/slurm_parallelcluster_queue_partition.conf

```diff
@@ -0,0 +1,32 @@
+# This file is automatically generated by pcluster
+{% set has_node = namespace(has_node=false) %}
```

Comment:
is a namespace required to set a variable from a different scope or a normal variable is enough here?
We might want to change the name of the namespace if we need to keep it to something like ns so that we don't have something like `has_node.has_node`